app/jobs/autotune/project_job.rb

```diff
@@ -94,8 +94,8 @@ def perform(project, update: false, target: :preview, current_user: nil)
       project.file_unlock!
       project.save!
 
-      project.build(current_user) if project.repeat_build?
-    rescue => exc
+      project.build(current_user, :publish => target == :publish) if project.repeat_build?
+    rescue StandardError => exc
```

Comment:
Would StandardError rescue catch exception raised in line 50? My understanding is that a raise will raise type Exception which is the base type of StandardError. Could end up leaving the lock on